### PR TITLE
Create fabric.rb

### PR DESCRIPTION
**Pull request by AndrewSB**
related https://github.com/caskroom/homebrew-cask/pull/17319

### CODE REVIEW

File: Casks/fabric.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'fabric' do
+  version :latest
+  sha256 :no_check
+
+  # fabric.io is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/fabric.rb]: Comment must go. It should only be present when maintainers can verify it, and in this case we can’t.

**Comment by AndrewSB** [comment]
made the change

File: Casks/fabric.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'fabric' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://ssl-download-crashlytics-com.s3.amazonaws.com/fabric/builds/Fabric-latest.zip'
+  appcast 'https://ssl-download-crashlytics-com.s3.amazonaws.com/fabric/version.xml',
+          :sha256 => '261546b7f617365ffa215ed1d1184f6892ce162ab035a3f31dee90058ef8fa9a'
```
Comment (vitorgalvao) [inline Casks/fabric.rb]: This no longer exists (which means it’s likely also being calculated wrongly). [Documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#appcast-stanza-details).

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by AndrewSB** [comment]
removed the hash as well. Squashing now

**Comment by AndrewSB** [comment]
@vitorgalvao squashed

**Pull request merged**